lib/cli/scripts/generate-sb-packages-versions.js

```diff
@@ -40,12 +41,19 @@ const run = async () => {
     .sort((package1, package2) => package1.name.localeCompare(package2.name))
     .reduce((acc, { name }) => ({ ...acc, [name]: updatedVersion }), {});
 
+  const prettierConfig = await prettier.resolveConfig(
+    path.join(__dirname, '..', '..', '..', '.prettierrc')
+  );
+
   await writeFile(
     path.join(__dirname, '..', 'src', 'versions.ts'),
-    dedent`
+    prettier.format(
+      dedent`
       // auto generated file, do not edit
       export default ${JSON.stringify(packageToVersionMap, null, 2)}
-    `
+    `,
+      { ...prettierConfig, parser: 'typescript' }
+    )
   );
```

Comment:
alternatively we could run `eslint <file-path> --fix`

I'd also propose we move the script to the root of the monorepo.

Reply:
Done!